WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/WatchDog.java

```diff
@@ -22,8 +23,15 @@
 import nl.tudelft.watchdog.intellij.util.WatchDogUtils;
 import org.jetbrains.annotations.NotNull;
 
+import javax.swing.*;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+import java.awt.event.WindowFocusListener;
+
 public class WatchDog implements ProjectComponent {
```

Comment:
This class is the "main" class of our IntelliJ plugin and our plugin is on every IntelliJ instance registered under this name (see https://github.com/levaja/watchdog/blob/master/WatchDogIntelliJPlugin/WatchDog/resources/META-INF/plugin.xml#L55). I suggest to keep this name.